Userland/Libraries/LibGUI/TabWidget.h

```diff
@@ -105,6 +109,7 @@ class TabWidget : public Widget {
     Function<void(Widget&)> on_tab_close_click;
     Function<void(Widget&, ContextMenuEvent const&)> on_context_menu_request;
     Function<void(Widget&)> on_double_click;
+    Function<void(Widget&, bool const previous, bool const modified)> on_modified_update;
```

Comment:
similarly, `on_tab_modified_update` or ... do we have other callbacks with a similar purpose? `on_tab_modification_change` ?

Reply:
I've removed this callback, as it wasn't actually being used